docs/api-usage/authentication.mdx

```diff
@@ -76,25 +76,34 @@ A decoded token has the following structure:
 }
 ```
 
-You can verify tokens in your code by checking that it's signed with a trusted key. You can find the necessary JSON Web Key Set (JWKS) by visiting `https://<your-saleor-domain>/.well-known/jwks.json`.
+### Refresh token structure
 
-### Verifying the access token
+Saleor's refresh tokens are [JSON Web Tokens](https://jwt.io) signed with RS256.
 
-To verify a token, use the [`tokenVerify`](api-reference/mutations/token-verify.mdx) mutation:
+A decoded token has the following structure:
 
-```graphql {2}
-mutation {
-  tokenVerify(token: "<token>") {
-    isValid
-    errors {
-      field
-      code
-    }
-  }
+```json
+{
+  "iat": 1671039993,
+  "owner": "saleor",
+  "iss": "http://demo.saleor.io/graphql/",
+  "exp": 1673631993,
+  "token": "K3IJEFFZgshc",
+  "email": "admin@example.com",
+  "type": "refresh",
+  "user_id": "VXNlcjo1OQ==",
+  "is_staff": true,
+  "csrf_token": "cAgshsjNr9AnsMKsAkpq4Q3fb4qBBPQT5mIyDMxShQEyO1fVpwCVzVyNQAJAZfKi"
 }
 ```
 
-You can also use the [`me`](api-reference/queries/me.mdx) query to retrieve information about the currently authenticated user:
+:::info
+`csrf_token` field is deprecated and is not used in current authentication flow.
+:::
+
+### Fetching information about currently authenticated user
```

Comment:
```suggestion
### Fetching information about the current user
```

IMO shorter = easier to parse in the navigation

Reply:
I specifically used the term `currently authenticated user` because it was already used in several other places on this page.